example/lib/main.dart

```diff
@@ -5,7 +5,7 @@ import 'package:flutter_dotenv/flutter_dotenv.dart' as dotenv;
 Future main() async {
   await dotenv.load(mergeWith: {
     'TEST_VAR': '5',
-  }); //mergeWith optional, accept Platform.environment for Mobile/Desktop APp
+  }); // mergeWith optional, accept Platform.environment for Mobile/Desktop app
```

Comment:
Do you mean except instead of accept?

Reply:
I believe the accept means the load function accepts Platform.environment as the mergeWith variable in the case of Mobile and Desktop platforms, and not for web.

Reply:
changed to: `mergeWith optional, you can include Platform.environment for Mobile/Desktop app`